lib/WebpackOptionsDefaulter.js

```diff
@@ -193,6 +193,7 @@ class WebpackOptionsDefaulter extends OptionsDefaulter {
 			options => (isProductionLikeMode(options) ? "warning" : false)
 		);
 
+		this.set("optimization", "call", value => Object.assign({}, value));
```

Comment:
Why does this one need `Object.assign({}, value)` instead of just defaulting to `{}` like its subkeys? Or is that something that is needed only on all top-level objects?

Reply:
> Or is that something that is needed only on all top-level objects?

Yep, it basically clones all modified objects